{
  "cve": {
    "data_type": "CVE",
    "data_format": "MITRE",
    "data_version": "4.0",
    "CVE_data_meta": {
      "ID": "CVE-2023-30991",
      "ASSIGNER": "psirt@us.ibm.com"
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": [

          ]
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "url": "https://www.ibm.com/support/pages/node/7047499",
          "name": "https://www.ibm.com/support/pages/node/7047499",
          "refsource": "MISC",
          "tags": [

          ]
        },
        {
          "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/254037",
          "name": "https://exchange.xforce.ibmcloud.com/vulnerabilities/254037",
          "refsource": "MISC",
          "tags": [

          ]
        }
      ]
    },
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "IBM Db2 for Linux, UNIX and Windows (includes Db2 Connect Server) 11.1 and 11.5 is vulnerable to denial of service with a specially crafted query.  IBM X-Force ID:  254037."
        }
      ]
    }
  },
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": [

    ]
  },
  "impact": {
  },
  "publishedDate": "2023-10-16T23:15Z",
  "lastModifiedDate": "2023-10-17T12:38Z"
}